dynamodb-stream-reader/src/main/java/no/unit/nva/dynamodb/DynamoDBEventTransformer.java

```diff
@@ -1,27 +1,30 @@
 package no.unit.nva.dynamodb;
 
 import com.amazonaws.services.lambda.runtime.events.models.dynamodb.AttributeValue;
-import elasticsearch.ElasticSearchIndexDocument;
+import no.unit.nva.elasticsearch.Constants;
+import no.unit.nva.elasticsearch.ElasticSearchIndexDocument;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
-import static no.unit.nva.dynamodb.DynamoDBStreamHandler.CONTRIBUTORS_IDENTITY_NAME;
-import static no.unit.nva.dynamodb.DynamoDBStreamHandler.DATE_YEAR;
-import static no.unit.nva.dynamodb.DynamoDBStreamHandler.DESCRIPTION_MAIN_TITLE;
-import static no.unit.nva.dynamodb.DynamoDBStreamHandler.PUBLICATION_TYPE;
+import static no.unit.nva.elasticsearch.Constants.CONTRIBUTORS_IDENTITY_NAME;
+import static no.unit.nva.elasticsearch.Constants.CREATED_DATE_KEY;
+import static no.unit.nva.elasticsearch.Constants.DATE_YEAR;
+import static no.unit.nva.elasticsearch.Constants.DESCRIPTION_MAIN_TITLE;
+import static no.unit.nva.elasticsearch.Constants.MODIFIED_DATE_KEY;
+import static no.unit.nva.elasticsearch.Constants.OWNER_NAME_KEY;
```

Comment:
We need to take a deeper look at this class in a new round

Reply:
Yess..